build/Dockerfile

```diff
@@ -11,4 +13,5 @@ ENV OPERATOR_PATH=/gcp-project-operator \
 WORKDIR /root/
 COPY --from=builder /gcp-project-operator/build/_output/bin/${OPERATOR_BIN} /usr/local/bin/${OPERATOR_BIN}
 LABEL io.openshift.managed.name="gcp-project-operator" \
-      io.openshift.managed.description="This operator will be responsible for managing GCP project and credentials."
+    io.openshift.managed.description="This operator will be responsible for managing GCP project and credentials."
```

Comment:
is that better whitespace wize? or was that by your editor

if it was by your editor, we should card later (for boilerplate) to lint the dockerfiles for a standard format so we don't need to review on this even

Reply:
It was my editor (VScode).
Lint checks on Dockerfiles sounds good to me, will have it carded after discussing it with FL.